src/test/java/org/protege/editor/owl/server/http/handlers/CodeGenHandlerTest.java

```diff
@@ -0,0 +1,30 @@
+package org.protege.editor.owl.server.handlers;
+
+import junit.framework.TestCase;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.protege.editor.owl.server.http.handlers.CodeGenHandler;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.when;
+
+public class CodeGenHandlerTest extends TestCase {
+
+    @Test
+    public void generateNTokens() {
+        List<String> codes = CodeGenHandler.generateCodes(10, 5, "p", "s", "_");
```

Comment:
I'd add a test also for the garden path, which is a prefix "C", no suffix (null) and no delimiter (null)